taskcluster/config.yml

```diff
@@ -18,46 +18,48 @@ taskgraph:
 # The list of valid stages that can be used with `target-stage and `start-stage`.
 # These get attached to tasks in `kinds`.
 valid-stages:
-    - clean-corpus
-    - clean-mono
+    - alignments-backtranslations
+    - alignments-distillation
+    - alignments-teacher
```

Comment:
I see you've split the alignments stage, but "alignments-teacher" looks inconsistent with other naming. "alignments-backtranslations" is also used to train the teacher. Maybe "alignments-parallel" would be more consistent? Of it, it's still ambiguous, something like "alignments-teacher-backtranslations" and "alignments-teacher-parallel".